data-prepper-api/src/main/java/org/opensearch/dataprepper/model/configuration/SinkModel.java

```diff
@@ -70,21 +76,27 @@ private static class SinkInternalJsonModel extends InternalJsonModel {
         @JsonProperty("routes")
         private final List<String> routes;
 
+        @JsonInclude(JsonInclude.Include.NON_EMPTY)
+        @JsonProperty("tags_key_name")
```

Comment:
I don't think we use the term `key_name` much. I think just calling this `tags_key` would be consistent. Or perhaps `tags_target_key`?